packs/data/feats.db/ferocious-gust.json

```diff
@@ -13,7 +13,7 @@
             "value": 2
         },
         "description": {
-            "value": "<p><strong>Frequency</strong> once per 10 minutes</p>\n<hr />\n<p>With heavy wing beats, you whip up a furious gust and direct it at your opponents. This air blast has the effects of <em>@Compendium[pf2e.spells-srd.Gust of Wind]{Gust of Wind}</em> with a DC equal to your class DC or spell DC, whichever is higher.</p>"
+            "value": "<p><strong>Frequency</strong> once per 10 minutes</p>\n<hr />\n<p>With heavy wing beats, you whip up a furious gust and direct it at your opponents. This air blast has the effects of <em>@Compendium[pf2e.spells-srd.Gust of Wind]{Gust of Wind}</em> with a @Check[type:fortitude|dc:resolve(@actor.attributes.classOrSpellDC.value)]{DC} equal to your class DC or spell DC, whichever is higher.</p>"
```

Comment:
I'd rather have the button but explicit in being a fortitude check, instead of just saying DC. So just add it at the end with a parenthetical. I'd make the suggestion but there's a bug in github that breaks suggestions with `@` in them.